lift/build.gradle

```diff
@@ -21,6 +21,7 @@ dependencies {
   compile("com.github.scopt:scopt_" + scalaVersionShort + ":3.5.0")
   compile("org.apache.spark:spark-core_" + scalaVersionShort + ":" + sparkVersion)
   compile("org.apache.spark:spark-sql_" + scalaVersionShort + ":" + sparkVersion)
+  compile("org.apache.spark:spark-mllib_" + scalaVersionShort + ":" + sparkVersion)
```

Comment:
Can you also add this to the DEPENDENCIES.md doc?

Reply:
Isn't this already covered by "Apache Spark and its subcomponents:" in the DEPENDENCIES.md doc?

Reply:
Ah yes, I missed that. This should be included. Thanks for catching that.